main.go

```diff
@@ -51,6 +64,66 @@ func (c *runCmd) Run() error {
 	return nil
 }
 
+func (c *fmtCmd) Run() error {
+	if len(c.Files) == 0 {
+		return format(os.Stdin, os.Stdout, c.Check)
+	}
+	var out io.StringWriter = os.Stdout
+	for _, filename := range c.Files {
+		in, err := os.Open(filename)
+		if err != nil {
+			return err
+		}
+		if c.Write {
+			out, err = os.CreateTemp("", "evy")
+			if err != nil {
+				return fmt.Errorf("%s: %w", filename, err)
+			}
+		}
+		if err := format(in, out, !c.Write); err != nil {
```

Comment:
should `!c.Write` be `c.Check` ? You should be able to run `evy fmt foo.evy` and it will write the output to stdout. This is what `goimports` does